packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -291,7 +292,8 @@ export const CollectionScreenDetailsTile = ({
       hideListenCount={true}
       hasStreamAccess={hasStreamAccess}
       streamConditions={streamConditions}
-      hideRepost={hideRepost || !isReachable}
+      hideFavorite={hideFavorite || !hasStreamAccess}
+      hideRepost={hideRepost || !isReachable || !hasStreamAccess}
```

Comment:
why use `isReachable` for repost but not for favorite? Shouldn't they be the same?

Reply:
Actually, you are allowed to unfavorite things while offline to remove them from your library and downloads

Reply:
and you shouldn't be able to access the collection page while offline unless it's been downloaded and is already part of your library